stable_baselines3/common/buffers.py

```diff
@@ -132,8 +132,10 @@ def to_torch(self, array: np.ndarray, copy: bool = True) -> th.Tensor:
         :return:
         """
         if copy:
-            return th.tensor(array).to(self.device)
```

Comment:
as this PR is about optimization, could you give `copy=False` as default a try?
(both CPU/GPU and check quickly that it doesn't change the results)

Reply:
I also tried `non_blocking=True` but it didn't work, probably because I was using `to()`?

https://github.com/DLR-RM/stable-baselines3/commit/6b147d8cca54e525d960d6d076bc7bc30cb1c73f